x-pack/examples/reporting_example/public/components/app.tsx

```diff
@@ -78,17 +94,64 @@ export const ReportingExampleApp = ({
     };
   };
 
+  const getPDFJobParamsDefaultV2 = (): JobParamsPDFV2 => {
+    return {
+      version: '8.0.0',
+      layout: {
+        id: constants.LAYOUT_TYPES.PRESERVE_LAYOUT,
+        selectors: getDefaultLayoutSelectors(),
```

Comment:
Please remove the selectors field. See: https://github.com/elastic/kibana/issues/107467